src/test/java/uk/gov/companieshouse/web/pps/controller/pps/BankTransferPenaltyReferenceControllerTest.java

```diff
@@ -78,25 +88,28 @@ void postRequestBankTransferPenaltyReferenceEmptySelected() throws Exception {
     @Test
     @DisplayName("Post PPS Bank Transfer Which Penalty Reference - late filing selection")
     void postRequestBankTransferPenaltyReferenceLateFilingSelected() throws Exception {
-        this.mockMvc.perform(post(PPS_BANK_TRANSFER_WHICH_PENALTY_PATH)
-                        .param(PPS_SELECTED_PENALTY_REF_ATTR, PenaltyReference.LATE_FILING.getPenaltyReference()))
+        this.mockMvc.perform(post(penaltyConfigurationProperties.getBankTransferWhichPenaltyPath())
+                        .param(PPS_SELECTED_PENALTY_REF_ATTR, PenaltyReference.LATE_FILING.getStartsWith()))
                 .andExpect(model().errorCount(0))
                 .andExpect(status().is3xxRedirection())
-                .andExpect(view().name("redirect:" + PPS_BANK_TRANSFER_LATE_FILING_PATH));
+                .andExpect(view().name("redirect:"
+                        + penaltyConfigurationProperties.getBankTransferLateFilingDetailsPath()));
     }
 
     @Test
     @DisplayName("Post PPS Bank Transfer Which Penalty Reference - sanctions selection")
     void postRequestBankTransferPenaltyReferenceSanctionSelected() throws Exception {
-        this.mockMvc.perform(post(PPS_BANK_TRANSFER_WHICH_PENALTY_PATH)
-                        .param(PPS_SELECTED_PENALTY_REF_ATTR, PenaltyReference.SANCTIONS.getPenaltyReference()))
+        this.mockMvc.perform(post(penaltyConfigurationProperties.getBankTransferWhichPenaltyPath())
+                        .param(PPS_SELECTED_PENALTY_REF_ATTR, PenaltyReference.SANCTIONS.getStartsWith()))
                 .andExpect(model().errorCount(0))
                 .andExpect(status().is3xxRedirection())
-                .andExpect(view().name("redirect:" + PPS_BANK_TRANSFER_SANCTIONS_PATH));
+                .andExpect(view().name("redirect:"
+                        + penaltyConfigurationProperties.getBankTransferSanctionsPath()));
     }
 
     private void configurePreviousController() {
         when(mockNavigatorService.getPreviousControllerPath(any()))
                 .thenReturn(MOCK_CONTROLLER_PATH);
     }
+
```

Comment:
Should this whitespace be removed?

Reply:
No, I think it's fine to keep this line for readability